build/ci/phase-template.yml

```diff
@@ -19,15 +19,19 @@ phases:
           Build_Debug:
             _configuration: Debug
             _config_short: D
+            _includeBenchmarkData: false
           Build_Release:
             _configuration: Release
             _config_short: R
+            _includeBenchmarkData: true
```

Comment:
>_includeBenchmarkData [](start = 12, length = 21)

Does this always enable the benchmarking when building on release builds? Main concern is that I dont want to run benchmarks when I do a release build on my machine.

Reply:
This downloads the data required for some of the benchmarks. It’s nothing has to do with running benchmarks.
The version of the benchmarks that we have enabled for release builds is a dry job version. Dry job version refers running these benchmarks just once. 
Currently we have just enabled dry job for benchmarks which doesn’t take more couple of seconds to run
If you want to actually run the benchmarks, then you will have to go into the benchmarks project and run and they won’t be run automatically

Reply:
OK thanks for the explanation.

---
In reply to: [246990441](https://github.com/dotnet/machinelearning/pull/2109#discussion_r246990441) [](ancestors = 246990441)

Reply:
Actually -- I just saw that the size of the datasets are 6.5 GB that we download from azure. In total how big is the download from Azure and how long does the download take? Does it always download or does it skip if the files are present?

---
In reply to: [247217818](https://github.com/dotnet/machinelearning/pull/2109#discussion_r247217818) [](ancestors = 247217818,246990441)

Reply:
It skips the files if they are already present.
It takes about 1.30 minutes to download the datasets on CI